src/NexusMods.App.UI/Pages/LoadoutGrid/LoadoutGridViewModel.cs

```diff
@@ -197,7 +197,7 @@ public LoadoutGridViewModel() : base(null!)
 
     public Task AddModAdvanced(string path)
     {
-        var installer = _provider.GetKeyedService<IModInstaller>("AdvancedInstaller");
+        var installer = _provider.GetKeyedService<IModInstaller>("AdvancedManualInstaller");
```

Comment:
we should probably make this a constant and use that on both ends (registration and usage) so we don't get this again.

Reply:
I would prefer to do that too. I actually tried doing that.

The issue is, the `DataModel` refers to `AdvancedManualInstaller` key.
Using a constant name requires that I add a dependency to the DataModel.

The alternative is I define the constant in the DataModel.

Reply:
Edit: We discussed this in meeting